entity-framework/core/modeling/owned-entities.md

```diff
@@ -0,0 +1,81 @@
+---
+title: EF Core | Owned Entity Types | Microsoft Docs
+author: julielerman
+
+uid: core/modeling/owned-entities
+---
+# Owned Entity Types
+
+This feature was added to EF Core 2.0.
+
+Owned entity type allow you to map types that do not have their own identity and are used as properties in any of your entities (such as a value object). An owned entity type shares the same CLR type with another entity type. The entity containing the defining navigation is the owner. When querying the owner the owned types will be included by default.
```

Comment:
> types that do not have their own identity
I wonder if this should be reworded to be more accurate as owned types do have identity, but the owner navigation property is part of this identity. Though this might not be a point of confusion. @divega Opinion?
> An owned entity type shares the same CLR type with another entity type
Should be "An owned entity type can share the same CLR type with another owned entity type"